### ISSUE TITLE
Enhancement: Make the instrumental selection look better

### ISSUE THREAD

**Issue by MrMooey** [FunkinCrew/Funkin#3313]
### Issue Checklist

- [X] I have properly named the enhancement
- [X] I have checked the issues/discussions pages to see if the enhancement has been previously suggested

### What is your suggestion, and why should it be implemented?

I had an idea for a way to make the instrumental menu look a little nicer, because right now it's a little barebones. Basically instead of using regular text to show which instrumental you're using, you can reuse the stylized name art from the character select menu. This means that the menu will look a little nicer and you won't to make any special art for it, even for any new characters that are added.

I made a quick mockup to show what this would look like in-game
![coolinstchangeideaPICO](https://github.com/user-attachments/assets/8f05c5c3-82eb-4668-a81e-d4121f495637)
![coolinstchangeideaBF](https://github.com/user-attachments/assets/30b4deb3-0f62-4f27-9655-c6df4c91c587)

It should be implemented because the menu looks a little out of place compared to how polished every other menu looks, so unless there are plans to overhaul it in the future this would be an easy way to make it look nicer without doing extra work for it

**Comment by 7oltan** [comment]
i wish they would do something like psych engine where you press a button and it shows you a menu, instead of it just jumpscaring you. it would be alot for new players

**Comment by MrMooey** [comment]
> i wish they would do something like psych engine where you press a button and it shows you a menu, instead of it just jumpscaring you. it would be alot for new players

Yeah I think for the full game pressing something like ctrl to open a menu would be nice. Maybe adding some modifiers and preferences could be fun and adds some replayability to songs that would usually be pretty boring for higher level players. I think this menu though can stay as it is, I imagine that opening a menu manually would be a little too slow and feel clunky. I think the solution would be just to make this toggleable in options, or making it so that pressing something like shift+enter opens this menu and plays the song but just pressing enter just plays the song as normal.

**Comment by 7oltan** [comment]
> > i wish they would do something like psych engine where you press a button and it shows you a menu, instead of it just jumpscaring you. it would be alot for new players
> 
> Yeah I think for the full game pressing something like ctrl to open a menu would be nice. Maybe adding some modifiers and preferences could be fun and adds some replayability to songs that would usually be pretty boring for higher level players. I think this menu though can stay as it is, I imagine that opening a menu manually would be a little too slow and feel clunky. I think the solution would be just to make this toggleable in options, or making it so that pressing something like shift+enter opens this menu and plays the song but just pressing enter just plays the song as normal.

I don’t think it will be clunky I read the flixel repository like it’s a book and it shouldn’t normally be laggy (I say normally cuz the devs…) i might make a pr with the menu and add some stuff like resetting the song score changing the inst (already there) and maybe setting the stage??

**Comment by H1nataDev** [comment]
Also Make This Black BG Nicer
